src/DI.php

```diff
@@ -487,6 +487,11 @@ public static function introFactory()
 		return self::$dice->create(Contact\Introduction\Factory\Introduction::class);
 	}
 
+	public static function localRelationship(): Contact\LocalRelationship\Repository\LocalRelationship
```

Comment:
Do we really need this `DI::` call because of the Module constructor now?

Reply:
Good question, probably not.

Reply:
Done.